packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -160,6 +160,39 @@ When the `smsRole` property is specified, the `smsRoleExternalId` may also be sp
 assume role policy should be configured to accept this value as the ExternalId. Learn more about [ExternalId
 here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
 
+### Attributes
+
+Attributes represent the various properties of each user that's collected and stored in the user pool. Cognito
+provides a set of standard attributes that are available for all user pools. Users are allowed to select any of these
+standard attributes to be required. Users will not be able to sign up to the user pool without providing the required
+attributes. Besides these, additional attributes can be further defined, and are known as custom attributes.
+
+Learn more on [attributes in Cognito's
+documentation](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html).
+
+The following code sample configures a user pool with two standard attributes (name and address) as required, and adds
+four optional attributes.
+
+```ts
+new UserPool(this, 'myuserpool', {
+  // ...
+  // ...
+  attributes: {
+    required: [ StandardAttrs.address, StandardAttrs.name ],
```

Comment:
What was the motivation behind shortening `Attributes` with `Attrs`. Usually we try to avoid abbreviations ("props" is an intentional outlier, which I kind of regret).

Reply:
No deep motivation of any sort. Attrs is a common abbreviation for attributes, just like props for properties and config for configuration.

I can expand it, if you prefer.

Reply:
Let's expand. We generally try not avoid abbreviations